docs/epic/OPTIMIZED-PR-PLAN.md

```
@@ -87,99 +88,111 @@ Final Delivery:
-
-| Kriterium | Alter Plan (8 PRs) | Neuer Plan (4 PRs) |
-|-----------|-------------------|-------------------|
-| Review-Overhead | Hoch | Niedrig |
-| Context-Switching | Viel | Wenig |
-| Substanz pro PR | Gering | Hoch |
-| Release-Zyklen | Lang | Kurz |
-| Verständlichkeit | Komplex | Klar |
+## Empfohlene Reihenfolge
 
-**Goldilocks-Prinzip:** Nicht zu viele (Overhead), nicht zu wenige (Review unmöglich), sondern genau richtig.
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)